go/api/generic_commands.go

```diff
@@ -30,4 +30,45 @@ type GenericBaseCommands interface {
 	//
 	// [valkey.io]: https://valkey.io/commands/del/
 	Del(keys []string) (Result[int64], error)
+
+	// Serialize the value stored at key in a Valkey-specific format and return it to the user.
+	//
+	// Parameters:
+	//  The key to serialize.
+	//
+	// Return value:
+	//  The serialized value of the data stored at key
+	//  If key does not exist, null will be returned.
+	//
+	// Example:
+	//  result, err := client.Dump([]string{"key"})
+	//  if err != nil {
+	//      // handle error
+	//  }
+	//  fmt.Println(result.Value()) // Output: String
```

Comment:
Result of `dump` isn't printable. It is just a byte array, which cannot be converted to a UTF-8 string.

Reply:
Hi @Yury-Fridlyand,  I'm working on the **restore** command. I'll keep you updated

Reply:
Hi @Yury-Fridlyand, already push my restore command implementation. 
Commit: 7b4710b7643809fe5d8703c0f7cb8f5e942254a8